ansible/roles/loadbalancer/defaults/main.yml

```diff
@@ -105,6 +134,13 @@ keepalived_default_volumes:
   - "/lib/modules:/lib/modules:ro"
   - "{{ 'haproxy_socket:/var/lib/kolla/haproxy/' if enable_haproxy | bool else '' }}"
   - "{{ 'proxysql_socket:/var/lib/kolla/proxysql/' if enable_proxysql | bool else '' }}"
+haproxy_ssh_default_volumes:
+  - "{{ node_config_directory }}/haproxy-ssh/:{{ container_config_directory }}/:ro"
+  - "/etc/localtime:/etc/localtime:ro"
+  - "{{ '/etc/timezone:/etc/timezone:ro' if ansible_facts.os_family == 'Debian' else '' }}"
+  - "haproxy_socket:/var/lib/kolla/haproxy/"
+  - "{{ 'letsencrypt:/etc/letsencrypt' if enable_letsencrypt | bool else omit }}"
+  - "{{ 'letsencrypt_certificates:/etc/haproxy/certificates' if enable_letsencrypt | bool else omit }}"
```

Comment:
Might be worth checking if this is actually creating an `omit_glenalfgknsfaklnef` volume.

Reply:
Checked, it doesn't.